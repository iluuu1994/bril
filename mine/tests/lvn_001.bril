@main {
  a: int = const 1;
  b: int = const 1;
  c: int = add a b;
  d: int = add a b;
  print c;
  print d;
}
