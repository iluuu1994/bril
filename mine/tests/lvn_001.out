@main {
  c: int = const 2;
  print c;
  print c;
}
